src/modules/debug/dotnet.ts

```diff
@@ -0,0 +1,17 @@
+import { Runtime } from '../../clients/runtime'
+import { getIOContext } from '../utils'
+import { ManifestEditor } from '../../lib/manifest'
+import logger from '../../logger'
+
+export default async (debugInst: string) => {
+  const manifest = await ManifestEditor.getManifestEditor()
+  const { name, vendor, builders } = manifest
+  const { dotnet } = builders
+  if (!dotnet) {
```

Comment:
_nit_: This would be super rare, but in case `builders` is not defined in the manifest:
```suggestion
  if (!builders?.dotnet) {
```